luajit.krun

```diff
@@ -125,6 +125,7 @@ BENCHMARKS = {
 SKIP = [
     "capnproto_decode:GC64:*", # Sometime crashes with GC64
     "capnproto_decode:RaptorJIT:*", # Sometimes crashes as well
+    "moonscript:RaptorJIT:*",
```

Comment:
Why are we skipping this?

Reply:
RaptorJIT is statically linking the Lua shared library and binary Lua modules are linked to it as shared a library. Moonscript happens to use some binary Lua modules